src/main/java/conversion/BuildVuServlet.java

```diff
@@ -34,12 +34,43 @@
 import java.util.concurrent.TimeUnit;
 import java.util.logging.Logger;
 
+/**
+ * Provides an API to use BuildVu on its own dedicated app server. See
+ * {@link BaseServlet} for more detailed information on the api since this class
+ * is focused mostly on conversions.
+ */
 @WebServlet(name = "buildvu", urlPatterns = {"/buildvu"})
 @MultipartConfig
 public class BuildVuServlet extends BaseServlet {
 
     private static final Logger LOG = Logger.getLogger(BuildVuServlet.class.getName());
 
+    /**
+     * Converts given pdf file to html using BuildVu. Errors at this stage of
```

Comment:
The servlet also supports conversion of office files to HTML too (via LibreOffice - see convertToPDF method below)